<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFeatureDialog</class>
 <widget class="QDialog" name="SelectFeatureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Newsy</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Feature selection methods:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Feature extraction methods:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop words elimination</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TF-IDF</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stemming</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
